rust/crd/src/authentication/ldap.rs

```diff
@@ -183,27 +181,15 @@ impl DruidLdapSettings {
     pub fn main_container_commands(&self) -> Vec<String> {
         let mut commands = Vec::new();
 
-        let runtime_properties_file: String = format!("{RW_CONFIG_DIRECTORY}/{RUNTIME_PROPS}");
-        let internal_client_password = format!("$(echo ${ENV_INTERNAL_SECRET})");
-
-        commands
-                .push(format!("echo \"Replacing LDAP placeholders with their proper values in {runtime_properties_file}\""));
-        commands.push(format!(
-            r#"sed "s|{PLACEHOLDER_INTERNAL_CLIENT_PASSWORD}|{internal_client_password}|g" -i {runtime_properties_file}"# // using another delimiter (|) here because of base64 string
-        ));
-
         if let Some((ldap_bind_user_path, ldap_bind_password_path)) =
             self.ldap.bind_credentials_mount_paths()
         {
-            let ldap_bind_user = format!("$(cat {ldap_bind_user_path})");
-            let ldap_bind_password = format!("$(cat {ldap_bind_password_path})");
-
             commands.push(format!(
-                    r#"sed "s/{PLACEHOLDER_LDAP_BIND_USER}/{ldap_bind_user}/g" -i {runtime_properties_file}"#
-                ));
+                "export {ENV_LDAP_BIND_USER}=\"$(cat {ldap_bind_user_path})\""
```

Comment:
TIL: Actually, we should use something like `druid.segmentCache.locations=${file:UTF-8:/config/segment-cache-def.json}` if possible (see https://druid.apache.org/docs/latest/configuration/#configuration-interpolation)

Reply:
oh cool! I should've read the docs better haha

Reply:
https://github.com/stackabletech/druid-operator/pull/571/commits/9144aa2f9155c3a3833e4cd8544a0b249b27a2d5 
https://github.com/stackabletech/druid-operator/pull/571/commits/6f8f24177e2a83228ff9a729bd6072b87c950c31